terraform/aws/analytical-platform-data-production/airflow/iam-policies.tf

```
@@ -145,3 +146,108 @@ data "aws_iam_policy_document" "airflow_dev_cluster_autoscaler_assume_role_polic
     actions = ["sts:AssumeRole"]
   }
 }
+
+# flow log policy
+
+data "aws_iam_policy_document" "airflow_dev_flow_log_role_policy" {
+  statement {
+    sid    = ""
+    effect = "Allow"
+    actions = [
+      "logs:PutLogEvents",
+      "logs:DescribeLogStreams",
+      "logs:DescribeLogGroups",
+      "logs:CreateLogStream",
+      "logs:CreateLogGroup"
+    ]
+    resources = ["*"]
+  }
+
+}
+
+
+
+data "aws_iam_policy_document" "airflow_dev_flow_log_assume_policy" {
+  statement {
+    sid    = ""
+    effect = "Allow"
+    principals {
+      type = "Service"
+      identifiers = [
+        "vpc-flow-logs.amazonaws.com"
+      ]
+    }
+    actions = ["sts:AssumeRole"]
+    condition {
+      test     = "StringEquals"
+      variable = "aws:SourceAccount"
+      values   = ["593291632749"]
```

```suggestion
      values   = [var.account_ids["analytical-platform-data-production"]]
```
refer to variable for the account number in terraform.tfvars